app/renderer/js/main.ts

```diff
@@ -66,6 +66,8 @@ interface SettingsOptions {
 	loading?: AnyObject;
 }
 
+type ServerOrFunctionalTab = ServerTab | FunctionalTab;
```

Comment:
When i used something like this, i got an error saying ```ServerTab is a class, can't be used as a type```

Reply:
Yeah that was because typescript didn't know about the file; I though it was because we used `declare module *`

Reply:
Oh, great!